### DISCUSSION TITLE
photography/photographs-took-in-hong-konng/

### DISCUSSION THREAD

**Discussion by giscus** [ecwu/ecwu.github.io#13] [Blog Post Discussion]
# photography/photographs-took-in-hong-konng/

Homepage and Blog of ECWU

https://ecwuuuuu.com/photography/photographs-took-in-hong-konng/

**Comment by forecast-star** [comment]
Those colors are very nice! Did you change it digitally?

**Reply by ecwu** [reply]
Thank you so much for your comment! I'm glad you like the colors in my photos. Yes, I do use digital editing software to enhance my images. For most of my released photo work, I use Adobe Lightroom or Capture One for retouching. In the editing process, I typically adjust the photo crop, lens perspective, and ensure the exposure and tone are correct.


When it comes to my landscape photos of Hong Kong Island, I apply a more sophisticated color adjustment technique that I like to call the "black-gold color tone." In simple terms, here's what I do:
- I shift the color hue for red, yellow, and green toward orange.
- I decrease the color saturation for green, blue, purple, and magenta.
- I slightly increase the brightness for orange, yellow, and red.


I hope this explanation clarifies the process for you. If you have any more questions, feel free to ask!

**Comment by JINXIAOYE** [comment]
Beautiful
